it/mail/chrome/messenger/accountCreation.properties

```diff
@@ -41,7 +41,7 @@ config_details_found=Sono stati rilevati i tuoi dettagli di configurazione!
 config_unverifiable=Impossibile verificare la configurazione - password o nome utente errati?
 incoming_found_specify_outgoing=Sono stati rilevati i dettagli di configurazione del server in entrata, specifica il nome host d'invio.
 outgoing_found_specify_incoming=Sono stati rilevati i dettagli di configurazione del server in uscita, specifica il nome host di ricezione.
-please_enter_missing_hostnames=Impossibile ipotizzare le impostazioni - si prega di inserire i nomi host mancanti.
+please_enter_missing_hostnames=Impossibile ipotizzare le impostazioni - si prega d'inserire i nomi host mancanti.
```

Comment:
Hi Sherman, 
The correct entry is "di inserire". 
Best, 
Andrea
Via InoMail on Android-------- Messaggio originale --------24 agosto 2016 03:52:52 CEST, Sherman Dickman <notifications@github.com> ha scritto:
In it/mail/chrome/messenger/accountCreation.properties:
> @@ -41,7 +41,7 @@ config_details_found=Sono stati rilevati i tuoi dettagli di configurazione!
>  config_unverifiable=Impossibile verificare la configurazione - password o nome utente errati?
>  incoming_found_specify_outgoing=Sono stati rilevati i dettagli di configurazione del server in entrata, specifica il nome host d'invio.
>  outgoing_found_specify_incoming=Sono stati rilevati i dettagli di configurazione del server in uscita, specifica il nome host di ricezione.
> -please_enter_missing_hostnames=Impossibile ipotizzare le impostazioni - si prega di inserire i nomi host mancanti.
> +please_enter_missing_hostnames=Impossibile ipotizzare le impostazioni - si prega d'inserire i nomi host mancanti.

@fenderman79  Do you mean:

diinserire or di inserire?

—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/postbox/localizations","title":"postbox/localizations","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/postbox/localizations"}},"updates":{"snippets":[{"icon":"PERSON","message":"@shermandickman in #26: @fenderman79  Do you mean:\r\n\r\ndiinserire or di inserire?"}],"action":{"name":"View Pull Request","url":"https://github.com/postbox/localizations/pull/26/files/63340a85f6ae3d0662a2998ad4efa42b76afe2a3#r75983453"}}}